feedr-starter-code/js/app.js

```diff
@@ -1,3 +1,195 @@
 /*
   Please add all Javascript code to this file.
-*/
\ No newline at end of file
+*/
+
+$( document ).ready(function() {
+
+function firstSourceFunction() {
+$.get("https://newsapi.org/v2/top-headlines?country=sa&apiKey=7482da6bb90a4055b3a214402af13c5a", 
```

Comment:
It's better if you put the APIs keys in an external file and not push it to Github!